egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -513,10 +586,14 @@ def forward(
             hx = (h_zeros, c_zeros)
 
         self.check_forward_args(input, hx, None)
+
+        flat_weights = self._get_flat_weights()
+        input, *flat_weights = self.grad_filter(input, *flat_weights)
+
         result = _VF.lstm(
             input,
             hx,
-            self._get_flat_weights(),
+            flat_weights,
```

Comment:
should probably be including self.bias with the flat_weights.

Reply:
`_get_flat_weights()` returns all parameters including weights and bias.

Reply:
ah, bias is a bool, I guess